package.json

```diff
@@ -824,6 +824,11 @@
                                 "type": "boolean",
                                 "default": false,
                                 "description": "Delete any currently installed dev channel before starting the debug session"
+                            },
+                            "enableRemoteControl": {
+                                "type": "boolean",
```

Comment:
Let's expand this _slightly_ to accomodate some of the options described in the ticket.
![image](https://github.com/rokucommunity/vscode-brightscript-language/assets/2544493/4267bb34-224d-42cb-a007-5b68344f36ee)

```
 "remoteControlMode": {
    "oneOf": [
        {
            "type": "object",
            "description": "Options for activating and deactivating remote control mode",
            "properties": {
                "activateOnSessionStart": {
                    "type": "boolean",
                    "description": "Activate remote control mode on debug session start"
                },
                "deactivateOnSessionEnd": {
                    "type": "boolean",
                    "description": "Deactivate remote control mode on session end"
                }
            }
        },
        {
            "type": "boolean",
            "description": "Activate on session start, deactivate on session end."
        }
    ]
}
```